jekyll/_cci2/language-android.md

```diff
@@ -251,3 +251,8 @@ and may also increase memory usage.
 ## Deployment
 
 See the [Deploy]({{ site.baseurl }}/2.0/deployment-integrations/) document for examples of deploy target configurations.
+
+## Troubleshooting
+
+See the [Debugging Java OOM errors]({{ site.baseurl }}/2.0/oom/) document
```

Comment:
should the link be `/oom` if it's java-specific ? we might want a more general oom page in the future (if we don't already have one)—i'd suggest `/java-oom`

Reply:
Ah, great point — not only for this case but all other cases where we're migrating something from 1.0 Land.